presto-main/src/test/resources/catalog.json

```diff
@@ -0,0 +1,40 @@
+{
+  "catalogs": [
+    {
+      "user": "admin",
+      "catalog": ".*",
+      "allow": true
+    },
+    {
+      "catalog": "secret",
+      "allow": false
+    },
+    {
+      "user": ".*",
+      "catalog": "open-to-all",
+      "allow": true
+    },
+    {
+      "catalog": "all-allowed",
+      "allow": true
+    },
+    {
+      "user": "alice",
+      "catalog": "alice-catalog",
+      "allow": true
+    },
+    {
+      "user": "bob",
+      "catalog": "alice-catalog",
+      "allow": false
+    },
+    {
+      "catalog": "allowed-absent"
+    },
+    {
+      "user": "ÎÎÎ",
```

Comment:
Don't use unicode characters in the Presto codebase (sometimes they don't render right) -- use the \u values instead.